apps/scraper-runner/src/app/scraper-main.ts

```diff
@@ -0,0 +1,108 @@
+import * as scholarsScrape from '@foodmedicine/scholars-scraper';
```

Comment:
This file was directly copy and pasted from apps/scraper-runner/src/main.ts,
nothing else was changed

Reply:
Oh, except the options ```
opts?: {
  limit?: number;
  numberOfArticles?: number;
  scoreCutOff?: number;
}``` were added in